modules/creating-multiple-ingress-through-single-alb.adoc

```diff
@@ -73,67 +73,74 @@ $ oc create -f sample-single-lb.yaml
 apiVersion: networking.k8s.io/v1 <1>
```

Comment:
1. The end `<1>` should be removed.
2. Before creating ingress, should we create AWSLoadBalancerController and specify `spec.ingressClass: single-lb` ?
```
kind: AWSLoadBalancerController
apiVersion: networking.olm.openshift.io/v1
metadata:
  name: cluster
spec:
  ingressClass: single-lb
```

Reply:
> Before creating ingress, should we create AWSLoadBalancerController and specify 

Yes. After the creation of `IngressClassParams` and `IngressClass` and before the creation of the ingresses.

Reply:
Done.

Reply:
> The end <1> should be removed.

@kowen-rh : what you think? I don't quite see a common convention, sometimes the API version has a bullet point something not.

Reply:
@alebedev87 I'm not aware of a particular convention for API version callouts. I believe that @lihongan is correct that we should remove it here, namely because the callout now refers to the Ingress name, not the API version. The simplest fix is to just remove the first instance of `<1>` here.

Reply:
`<1>` removed.

Reply:
Sorry for not being clear. I'm asking to remove it because I see duplicated `<1>` in this  example, see
```
apiVersion: networking.k8s.io/v1 <1>             --->  first <1>
kind: Ingress
metadata:
  name: example-1 <1>                           ---> second <1>
  annotations:

```

Reply:
No, it's me, I mixed `IngressClass` with `Ingress` and removed `<1>` for the former. Fixed `Ingress` indexing now.